irohad/torii/processor/query_processor.hpp

```diff
@@ -37,24 +37,16 @@ namespace iroha {
        * Register client query
        * @param query - client intent
        */
-      virtual void queryHandle(
-          std::shared_ptr<shared_model::interface::Query> qry) = 0;
+      virtual std::unique_ptr<shared_model::interface::QueryResponse>
+      queryHandle(const shared_model::interface::Query &qry) = 0;
       /**
        * Register client blocks query
        * @param query - client intent
```

Comment:
Not consistent with function argument's name. This problem arises in many places, please, rename variables in comments at least in functions you touched, such as this or the one upward, for example